rcsbapi/search/search_query.py

```diff
@@ -1094,6 +1098,142 @@ def __str__(self):
             raise ValueError("Illegal Operator")
 
 
+class NestedAttributeQuery(Group):
+    """
+    Represents a search query restricted to a single nested attribute group.
+
+    Ensures that all attributes in the query fall within a valid nested indexing context, and provides
+    consistent behavior for nesting logic. It wraps `AttributeQuery` objects and restricts operations to those
+    allowed by the nested schema rules.
+
+    Args:
+        AttributeQuery (AttributeQuery): Two attribute-level queries that must all belong to the same nested context group.
```

Comment:
Can you actually make this two separate lines, to make it more clear/explicit that two `AttributeQuery` arguments are required?